draftjs_exporter/dom.py

```diff
@@ -37,23 +37,24 @@ def camel_to_dash(camel_cased_str):
         return dashed_case_str.replace('--', '-')
 
     @classmethod
-    def use(cls, engine=DOM_HTML5LIB):
+    def use(cls, engine=HTML5LIB):
         """
         Choose which DOM implementation to use.
         """
         if engine:
             if inspect.isclass(engine):
                 cls.dom = engine
             elif engine.lower() == cls.HTML5LIB:
-                cls.dom = DOM_HTML5LIB
+                cls.dom = import_string('draftjs_exporter.engines.html5lib.DOM_HTML5LIB')
             elif engine.lower() == cls.LXML:
-                from draftjs_exporter.engines.lxml import DOM_LXML
-                cls.dom = DOM_LXML
+                cls.dom = import_string('draftjs_exporter.engines.lxml.DOM_LXML')
             elif engine.lower() == cls.STRING:
-                from draftjs_exporter.engines.string import DOMString
-                cls.dom = DOMString
+                cls.dom = import_string('draftjs_exporter.engines.string.DOMString')
             else:
-                raise ConfigException('Invalid DOM engine.')
+                try:
+                    cls.dom = import_string(engine)
+                except ImportError:
+                    raise ConfigException('Invalid DOM engine.')
```

Comment:
I would have rather left the `ImportError` throw itself, but this would be a breaking change. If we only release this in 2.0.0 we can make this change though.